packages/ui/icons/comments/small.jsx

```diff
@@ -0,0 +1,19 @@
+import React, { PureComponent } from "react";
+
+
+export default class CommentsSmall extends PureComponent {
```

Comment:
@AlexBabusanu let's also add these new icons to the `stories/icons/icons.stories.mdx` file.

```

---
## Comments

import CommentsSmall from "@figshare/fcl/icons/comments/small";

<Story name="Comments (small)">
  <CommentsSmall className={styles.icon} />
</Story>

```jsx
import CommentsSmall from "@figshare/fcl/icons/comments/small";
```

```